docs/content/guides/7.multistore/3.patterns/5.subpath/1.subpath.md

```diff
@@ -0,0 +1,696 @@
+---
+
+# How to Make Sub-path Routing
+
+
+Sub-path routing allows you to serve different store configurations from the same Alokai deployment using URL paths. This approach is useful if the multibrand per-domain does not address your requirements.
+
+::info
+This guide focuses on implementing multi-store functionality through URL paths (like `/electronics`, `/apparel`). If domain-based separation is sufficient for your use case (like `electronics.example.com`, `apparel.example.com`), the [multibrand feature](https://docs.alokai.com/guides/multistore/tooling-and-concepts) is the preferred approach.
+::
+
+## Overview
+
+Sub-path routing in Alokai uses the [Config Switcher extension](https://docs.alokai.com/middleware/guides/config-switcher) with the header strategy. This allows your Middleware to dynamically select the appropriate configuration based on which store the user is browsing.
+
+For example, with this setup you could have:
+- `/en/electronics` - Electronics store with its specific configuration
+- `/en/apparel` - Apparel store with different products, styling, and configuration
+
+All of this is served from a single Frontend deployment.
+
+### How Config Switcher Works
+
+The Config Switcher dynamically changes your middleware integration configuration based on the current request configuration ID sent with the `x-alokai-middleware-config-id` header. This means:
+
+1. **Path-Based Configuration**
+    - Different configuration ID set by different paths allow to fetch data from different sources
+    - Example: `/apparel` path shows apparel products, while `/electronics` shows electronics products
+
+2. **Configuration ID vs Store ID**
+    - Each configuration has its own distinct ID
+    - This is different from the store ID (`storeId`)
+    - Multiple configurations can exist within a single store
+
+3. **Same-Store Operation**
+    - Config Switcher works within a single store instance
+    - It extracts the configuration ID from the request
+    - It executes requests using the appropriate configuration
+
+::tip Config Switcher Documentation
+This section provides a brief overview of setting up Config Switcher. For complete documentation, refer to the [Config Switcher guide](https://docs.alokai.com/middleware/guides/config-switcher).
+::
+
+### Key Distinctions To File-Based Inheritance
+
+`ConfigSwitcher` operates differently from file-based inheritance:
+
+| **File-Based Inheritance**              | **Config Switcher**                                              |
+| --------------------------------------- | ---------------------------------------------------------------- |
+| Works **between different stores**      | Works **within a single store**                                  |
+| Creates separate deployable instances   | Uses a single deployable instance                                |
+| Full codebase customization             | Only middleware configuration changes, Storefront code is shared |
```

Comment:
```suggestion
| Provides inheritance mechanism to make managing multiple stores easy             | You have to manage codebase reuse between stores on your own |
```
It's quite confusing, especially that we then proceed with customising UI per config switcher's store.